org.gtk.Gtk3theme.Yaru.json

```diff
@@ -57,14 +57,25 @@
         {
           "type": "git",
           "url": "https://github.com/ubuntu/yaru.git",
-          "commit": "5944aae75b448837da7032e6801848c514d9655a"
+          "commit": "r28"
         },
         {
           "type": "shell",
           "//": "Disable building everything not used",
           "commands": [
             "sed -i -e \"s/subdir('icons')//\" -e \"s/subdir('sounds')//\" -e \"s/subdir('gnome-shell')//\" -e \"s/subdir('sessions')//\" meson.build"
           ]
+        },
+        {
+            "type": "shell",
+            "//": "Replace gtk-3.0 folder with gtk-3.20",
```

Comment:
Your comment is backwards.

Reply:
It refers to the content of the folder (content of 3.0 is replaced with the content of 3.20), but I agree that it is unclear